### PR TITLE
Plastic is not a mineral anymore, plastic flaps can be constructed/deconstructed

### PR DESCRIPTION
**Pull request by uraniummeltdown**
Plastic is not /sheet/mineral but /sheet now, plastic flaps can be deconstructed with a screwdriver and welder (takes roughly 2x wall decon time) and made with 5 sheets of plastic.
Added a new file for construction defines

:cl:
add: Plastic flaps can be deconstructed with tools and constructed using 5 sheets of plastic.
/:cl:

### CODE REVIEW

**Comment by tigercat2000** [comment]
>plastic flaps can be constructed/deconstructed 

no
these are intentionally indestructable
see #6287

**Comment by uraniummeltdown** [comment]
We can just make them take even longer to get though and have a flat time (ignoring toolspeed), it's not like they are deleted you can just rebuild them.

**Comment by Markolie** [comment]
👍 Everything should be (de)constructible. It probably takes mote time to get through these + the windoor than an airlock or rwall.

**Comment by Birdtalon** [comment]
You can easily break down a wall or R-wall. Just make them take just as long to deconstruct as an R-Wall. Nothing should be invincible IMO except for Escape shuttle.

**Comment by Ssirius** [comment]
I'm also for this but the deconstruction timer should be longer then what it takes to get trough a normal wall, considering you wont need eye protection to take it down.

**Comment by FalseIncarnate** [comment]
Aside from a shorter path, is there any real change by removing the mineral subtype?

File: code/game/objects/items/stacks/sheets/sheet_types.dm [hunk 1]
```diff
@@ -289,3 +291,40 @@ var/global/list/datum/stack_recipe/runed_metal_recipes = list ( \
...
+	new/datum/stack_recipe("sucker mould", /obj/item/weapon/kitchen/mould/loli, 1, on_floor = 1), \
+	)
+
+/obj/item/stack/sheet/plastic
+	name = "plastic"
+	desc = "Compress dinosaur over millions of years, then refine, split and mold, and voila! You have plastic."
+	singular_name = "plastic sheet"
+	icon_state = "sheet-plastic"
+	throwforce = 7
+	flags = CONDUCT
```
Comment (davipatury) [inline code/game/objects/items/stacks/sheets/sheet_types.dm]: ...

**Comment by Lady-Luck** [comment]
plastic flaps having their own rules has bothered me for a long time. 👍

**Comment by Allfd** [comment]
My only problem with the plastic flaps was they made no sense and defied everything I thought I knew about strips of hanging plastic.

**Comment by Twinmold93** [comment]
They need to be (de)constructable. EVERYTHING should be. Hell, even the BSA is now buildable. They should take a bit of time, yes, and make quite a bit of noise in doing so. A time between regular walls and reinforced walls would be a good medium, since the ones that go into more restricted areas tend to, yes, have the windoor or reinforced window on the other side. (All the smartly made ones have only a windoor, which takes a while to break and makes one hell of a noise, alerting everyone nearby).

Just because these have always been 'immune' to configuration other than explosives doesn't mean they should remain as such.

**Comment by tigercat2000** [comment]
Eh. I've changed my opinion on this, after reading a bunch of other comments. It is better for them to be de/constructable, I suppose. However, they are far too easy to break right now- These are used mostly in areas that are completely surrounded in reinforced walls. They shouldn't be any easier to remove than that. 

That said, I linked to the issue because there were a fair few people argued against it; @Necaladun @Draconic-saint @Crazylemon64

**Comment by uraniummeltdown** [comment]
Yeah I'll increase the deconstruction time a bit

**Comment by Taaketa** [comment]
I belueve if the flaps are going to be deconstructable. The noise must be on par with that off forcing an offline door open. Then, as people have suggested, the difficulty of breaking into a department will be the same.

I really think there are only 3 departments that will suffer for this anyway. Cargo, Medical and RnD. All their flaps go ditect to their main access corridors. 

Sec, Engi, kitchen and Botany's flaps go to areas that are closed off by doors that'd need to be hacked anyway.

**Comment by Draconic-saint** [comment]
Still not a fan but I can understand the sentiment that everything should be buildable. 

I'd like to request that more tools are required than just two, making it sub-par to hacking and equal to deconstructing walls/r-walls in regards of things you need. Add in a welder, so it's not an easier way in if you can't find any goggles/masks/gloves for eye/shocking proctection. I'd like it for them to be a massive bitch to deconstruct so to make it very very obvious these are not "easy way in"

Don't rely on just sound as a deterent, have it be just as bad a challange as the existing options for possible burglars

**Comment by Necaladun** [comment]
I'm not against the idea - I like everything being deconstructable - it'll likely want some map edits though to make sure the security of the places remains intact.

**Comment by uraniummeltdown** [comment]
I'll change the wirecutters to a welder

**Comment by Anticept** [comment]
@Allfd indeed they are weird, but there isn't really a good... replacement. They also let mechs through.

**Pull request merged**